cupy/prof/time_range.py

```diff
@@ -17,6 +18,8 @@ def time_range(message, color_id=None, argb_color=None):
         message: Name of a range.
         color_id: range color ID
         argb_color: range color in ARGB (e.g. 0xFF00FF00 for green)
+        sync (bool): If ``True``, waits for completion of all outstanding
+            processings on GPU before calling RangePush() or RangePop()
```

Comment:
How about using ```:func:`cupy.cuda.nvtx.RangePush` ``` and ```:func:`cupy.cuda.nvtx.RangePop` ```

Reply:
All right, will update soon based on your comment.